NEWS.md

```diff
@@ -1,7 +1,7 @@
 ## v1.0.0 Released 2023-02-21
 ### APIs versions
 * Provides `instance-authority-links v1.0`
-* Provides `instance-authority-links-statistics v1.0`
+* Provides `instance-authority-links-statistics v2.0`
```

Comment:
This version already released. Please create a block for v2.0.0 and put this change there and also update the version of the module to v2.0.0